frontend/src/components/utilities/ResponsiveWrapper.vue

```diff
@@ -10,14 +10,7 @@ export interface Props {
   huge?: boolean;
 }
 
-const props = withDefaults(defineProps<Props>(), {
-  extraSmall: false,
-  small: false,
-  medium: false,
-  large: false,
-  extraLarge: false,
-  huge: false,
-});
+const props = defineProps<Props>();
```

Comment:
can we start renaming the props so they reflect the files they are declared for? It is super confusing if we are always exporting Props. If one wants to import several of them in one file it is predestined to create errors.

Reply:
Hmm, good point. What about `import { Props as ResponsiveWrapperProps} from …`?